packages/react-component-library/src/components/Timeline/types.ts

```diff
@@ -0,0 +1,21 @@
+export interface TimelineRootComponent {
```

Comment:
I have some discomfort about this pattern as if we need to add to the types of `children` that `Timeline` accepts then we need to make updates in multiple places.

Reply:
@thyhjwb6 added magic with generics!

🙌 🙌 🙌 🙌 🙌